### PR TITLE
Basic Layout of the Profile Page

### PR DESCRIPTION
**Pull request by emieczkowski**
I created the overall basic layout for the profile page, i.e. all of the labels, buttons, and icons/placement on the screen. I also created a basic model for data inputted into the profile, i.e. name, email, phone number, pronouns, and date joined, in a separate file called ProfileModel. 

Buttons do not yet do anything, and data is currently fake, i.e. just written inside the ViewController file. AppDelegate currently has the rootVC as the profile page for demoing, rather than the home page. 

What the profile page currently looks like:
Screen Shot 2019-11-05 at 6 09 04 PM

### CODE REVIEW

**Comment by SamChou19815** [comment]
Generally looks reasonable. Let's wait until #3 lands first. Then merge in latest master. Therefore we can have a more reviewable changeset.

**Review by SamChou19815 — APPROVED** [review]
Code looks good to me. Let @lsizemore8 make the final call.

**Review by lsizemore8 — APPROVED** [review]
Looks good to me as well

**Pull request merged**
